sections/resources/uas/UAS3Step.jsx

```diff
@@ -33,7 +33,11 @@ export const UAS3Step = () => (
       isCard={true}
     />
     <div>
-      <a href="https://discord.com/invite/vpyF7nCtkm">
+      <a
+        href="https://discord.com/invite/vpyF7nCtkm"
+        target="_blank"
+        rel="noreferrer noopener"
```

Comment:
Yes but I think this is outside the StudentResource component and not within as a props so it should be okay. The reason is so that it opens on a new tab and does not exit out of pump's website